internal/cmd/request.go

```diff
@@ -68,39 +63,15 @@ func SendCMDRequest(endpoint string, request any) (core.APIResponse, error) {
 	return response, nil
 }
 
-// Print the CMD response in it's expected format.
-// If in testing mode, print the CMD as a core.APIResponse type.
-func PrintCMDResponse(response core.APIResponse, responseType any) {
-	testingMode := config.TESTING_MODE.Get()
-
-	if !response.Success {
-		if testingMode {
-			fmt.Println(util.MustToJSONIndent(response))
-		} else {
-			fmt.Println(response.Message)
-		}
-
-		util.Exit(2)
-		return
-	}
-
-	if testingMode {
-		fmt.Println(util.MustToJSONIndent(response))
-		return
+func PrintCMDResponse(request any, response core.APIResponse, responseType any, verbose bool) {
```

Comment:
I would include a function description that includes something about printing the response content as JSON.

You should also describe the behavior of verbose. Just a quick note saying we will dump the entire request and response.